docs/source/guides/define_by_run.rst

```diff
@@ -0,0 +1,23 @@
+Define-by-run
```

Comment:
How about aligning the capitalization according to this page (capital _r_ in _run_)? https://docs.chainer.org/en/stable/tutorial/basic.html#define-by-run